charts/pomerium/README.md

```diff
@@ -215,6 +215,8 @@ A full listing of Pomerium's configuration variables can be found on the [config
 | `image.pullPolicy`                                           | Pomerium image pull policy                                                                                                                                                                                                                                                                         | `IfNotPresent`                                                              |
 | `service.annotations`                                        | Service annotations                                                                                                                                                                                                                                                                                | `{}`                                                                        |
 | `service.externalPort`                                       | Pomerium's port                                                                                                                                                                                                                                                                                    | `443` if `config.insecure` is `false`.  `80` if `config.insecure` is `true` |
+| `service.grpcTrafficPort.nameOverride`                       | Override name of grpc port in services.  Only use if required for protocol detection by mesh or ingress services                                                                                                                                                                                   | `https`/`grpc` in `secure`/`insecure` mode                                  |
+| `service.httpTrafficPort.nameOverride`                       | Override name of http port in services.  Only use if required for protocol detection by mesh or ingress services.  Set to `http2` for istio when offloading mtls to the mesh.                                                                                                                      | `https`/`http` in `secure`/`insecure` mode                                  |
```

Comment:
Maybe explicitly call out Istio here (or in the change notes?) for anyone cntrl+f'ing.

Reply:
For grpc, it actually defaults to being Istio friendly.  The override is in case other systems need something besides `grpc`.

Reply:
Added a note mentioning Istio in the Changelog so there's some clarity on the reasoning for the change.

Reply:
👍